app/models/article.rb

```diff
@@ -54,7 +54,9 @@ class Article < ApplicationRecord
            inverse_of: :commentable,
            class_name: "Comment"
 
-  validates :body_markdown, length: { minimum: 0, allow_nil: false }, uniqueness: { scope: %i[user_id title] }
+  validates :body_markdown, bytesize: { maximum: 800.kilobytes, too_long: "is too long." }
```

Comment:
the custom message is because by default Rails would use `too many characters` but it would confuse people as bytesize is not the same as characters, and nobody actually checks how many characters they have typed :D We could improve on the messaging but I think given its an edge case, we could live with this for the moment